src/components/ContactsPage.js

```diff
@@ -79,30 +93,83 @@ class ContactsPage extends React.Component {
     const updatedContacts = contacts.filter(x => !x.isChecked);
     this.setState({ contacts: updatedContacts });
   }
-  
-  numberOfSelectedContacts() {
+
+  numberOfSelectedContacts = () => {
     const { contacts } = this.state;
     return contacts.filter(x => x.isChecked).length || null;
   }
 
-  changeIsChecked(id) {
+  changeIsChecked = (id) => {
     const { contacts } = this.state;
     const contact = contacts.find(item => item.id === id);
     contact.isChecked = !contact.isChecked;
     this.setState({ contacts });
   }
 
-  updateLikes(id, step) {
+  updateLikes = (id, step) => {
     const { contacts } = this.state;
     contacts.find(item => item.id === id).likes += step;
     this.setState({ contacts });
   }
 
-  deleteContact(id) {
+  deleteContact = (id) => {
     const { contacts } = this.state;
     const updatedContacts = contacts.filter(c => c.id !== id);
     this.setState({ contacts: updatedContacts });
   }
+
+  deleteContactPopUp = (id) => {
```

Comment:
Add the 'open' keywoard at the beginning of all those functions that only open the popup `openDeleteContactPopUp`. A new guy seeing the code might think that those functions actually delete/clear contacts.